tests/utils.go

```diff
@@ -1805,135 +1806,140 @@ func GetContainerOfPod(pod *k8sv1.Pod, containerName string) *k8sv1.Container {
 }
 
 func cleanNamespaces() {
-	virtCli, err := kubecli.GetKubevirtClient()
+	virtCli, err := client.TestClientWithHighRateLimits()
 	util2.PanicOnError(err)
 
+	wg := sync.WaitGroup{}
+	wg.Add(len(TestNamespaces))
 	for _, namespace := range TestNamespaces {
+		go func(namespace string) {
+			defer wg.Done()
+			_, err := virtCli.CoreV1().Namespaces().Get(context.Background(), namespace, metav1.GetOptions{})
+			if err != nil {
+				return
+			}
 
-		_, err := virtCli.CoreV1().Namespaces().Get(context.Background(), namespace, metav1.GetOptions{})
-		if err != nil {
-			continue
-		}
-
-		// Clean namespace labels
-		err = libnet.RemoveAllLabelsFromNamespace(virtCli, namespace)
-		util2.PanicOnError(err)
+			// Clean namespace labels
+			err = libnet.RemoveAllLabelsFromNamespace(virtCli, namespace)
```

Comment:
nit: can we stick to the below style, i.e. just directly consuming the result of the action, or is there a reason for the declaration?